mem-03-D.xml

```diff
@@ -257,8 +263,8 @@ When used with {-lu':v}, this prefix means "Someone (indefinite subject) does so
       <column name="definition">be not attentive, be distracted, lack focus</column>
       <column name="definition_de">unaufmerksam, abgelenkt, unkonzentriert</column>
       <column name="definition_fa"></column>
-      <column name="definition_sv">vara ouppmärksam, vara distraherad, sakna koncentration</column>
```

Comment:
This is very frustrating, because you're submitting reversions of submitted commits again. I don't know what exactly you're doing to cause this to happen, but after the PR I sent you, you should've done all of your edits on top of that using it as a base. Instead, it looks like you've made your changes to an older commit, so that when you pull it into your master, you're reverting everything between that commit and what should've been your base.

The PR that you need to actually submit a change is from your master (after you've made changes on top) to De7vID/master. Does that make sense?

Reply:
@De7vID When i merged your last pull request in my master branch, i rebased this branch onto my master branch. Nothing more.